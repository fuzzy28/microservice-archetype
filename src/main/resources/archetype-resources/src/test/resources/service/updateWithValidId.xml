<?xml version="1.0" encoding="UTF-8"?>
<dataset>
	<employee propertyId="1" propertycode="Test" propertyname="Test" active="true" />
</dataset>