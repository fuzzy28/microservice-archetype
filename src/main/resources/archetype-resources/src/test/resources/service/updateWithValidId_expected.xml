<?xml version="1.0" encoding="UTF-8"?>
<dataset>
	<employee propertyId="1" propertyCode="Test Updated" propertyName="Test Updated" active="false"></employee>
</dataset>